src/web/app/pages-student/student-profile-page/upload-edit-profile-picture-modal/upload-edit-profile-picture-modal.component.ts

```diff
@@ -13,12 +14,19 @@ import { ImageCroppedEvent, ImageCropperComponent } from 'ngx-image-cropper';
 export class UploadEditProfilePictureModalComponent implements OnInit {
   imageChangedEvent: any = '';
   formData?: FormData;
+  isValidProfileFileType: boolean = false;
 
   @ViewChild(ImageCropperComponent) imageCropper!: ImageCropperComponent;
 
   @Input() image!: Blob | null;
 
-  constructor(public activeModal: NgbActiveModal) { }
+  constructor(public activeModal: NgbActiveModal,
+    private statusMessageService: StatusMessageService,
+    ) { }
+
+  get validProfileFileTypes(): string {
```

Comment:
You shouldn't need dedicated getter if the item to get is constant. Just a standard `readonly validProfileFileTypes: string = ...'` is sufficient.